webui/server/server.py

```diff
@@ -178,7 +179,7 @@ def Open(self):
       if self.ser:
         # Apply currently loaded thresholds when the microcontroller connects.
         for i, threshold in enumerate(self.profile_handler.GetCurThresholds()):
-          threshold_cmd = str(sensor_numbers[i]) + str(threshold) + '\n'
+          threshold_cmd = str(sensor_numbers[i]) + ' ' + str(threshold) + '\n'
```

Comment:
Since we're here can we change to using a format string (or f-string, or .format() -- whatever looks most readable to you):

```py
threshold_cmd = "%d %d\n" % (sensor_numbers[i], threshold)
```

Both here and below

Reply:
Oh yeah I forgot about this syntax. Done.